code/modules/mining/mine_items.dm

```diff
@@ -190,7 +190,7 @@
 		sleep(50)
 		var/clear = TRUE
 		for(var/turf/T in range(3,src))
```

Comment:
It IS 5 tiles long, and 5 wide. The create_room proc does actual perimeter stuff instead of radius/range like everything else.